frontend/src/pages/projects/screens/detail/data-connections/utils.ts

```diff
@@ -87,7 +87,7 @@ export const convertAWSSecretData = (dataConnection: DataConnection): AWSDataEnt
     .filter((key) => key !== AWS_KEYS.NAME)
     .map((key: AWS_KEYS) => ({
       key,
-      value: secretData?.[key] ? atob(secretData?.[key]) : '',
+      value: secretData[key] ? atob(secretData[key]) : '',
```

Comment:
Likely need to fix this function to first check if it's an AWS data connection as discussed in slack.